src/System.Net.Http/src/System/Net/Http/Managed/HttpProxyConnectionHandler.Unix.cs

```diff
@@ -0,0 +1,22 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+using System.IO;
+using System.Net.Http.Headers;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace System.Net.Http
+{
+    internal sealed partial class HttpProxyConnectionHandler : HttpMessageHandler
+    {
+        // On Unix we get default proxy configuration from environment variables
+        static private IWebProxy GetDefaultProxy()
```

Comment:
Nit, but I might call this something like "ConstructSystemProxy" just to make it clearer what it's going to do -- particularly that it's going to go read a bunch of config info.